packages/input-number/src/index.vue

```diff
@@ -103,14 +103,17 @@ export default defineComponent({
       type: Number,
       default: -Infinity,
     },
-    modelValue: Number,
+    modelValue: {
+      type: Number,
+      default: 0,
+    },
     disabled: {
       type: Boolean,
       default: false,
     },
     size: {
-      type: String as PropType<'large' | 'small'>,
-      validator: (val: string) => ['large', 'small'].includes(val),
+      type: String as PropType<'medium' | 'small' | 'mini'>,
+      validator: (val: string) => ['medium', 'small', 'mini'].includes(val),
```

Comment:
We had a global union type `ComponentSize` you can use it directly

Reply:
got it